CHANGELOG.rst

```diff
@@ -1,5 +1,11 @@
 Changelog for Mantarray Desktop App
 ===================================
+1.0.7 (unreleased)
+------------------
+
+Added:
```

Comment:
The format of this is incorrect. Please see the subheadings for other versions to how it should be done